.github/workflows/ci.yaml

```diff
@@ -52,6 +56,22 @@ jobs:
           inv generateapidocs
           inv checkchanges --action="run inv generateapidocs"
 
+      - name: Build and save MetalLB images
+        run: |
+          inv build --binaries all
+          docker save quay.io/metallb/speaker:dev-amd64 > speaker.tar
+          docker save quay.io/metallb/controller:dev-amd64 > controller.tar
+          docker save quay.io/metallb/mirror-server:dev-amd64 > mirror-server.tar
+
+      - name: Upload images
```

Comment:
Neat! Can we delete the artifacts as final step? I don't think we are gonna need those images for triaging failures.

Reply:
There isn't an official workflow for cleanup :confused: 
set the retention-days to 1 instead: https://github.com/actions/upload-artifact#retention-period

Reply:
Makes sense, thanks